homepage/tests/test_backend.py

```diff
@@ -1,5 +1,5 @@
-from backend import __version__
```

Comment:
I think the name of this file is off. from when we called the django app `backend`. Should be called something else probably.